WebDriverManager/DriverConfigs/Impl/FirefoxConfig.cs

```diff
@@ -46,6 +47,11 @@ public virtual string GetLatestVersion()
             }
         }
 
+        public virtual string GetMatchingBrowserInstalledVersion()
+        {
+            throw new NotImplementedException();
```

Comment:
Hmm, I awaited this to be thrown when using FF with `version: "matching_installed"` but it didn't. What could be the issue?